omnibus/config/projects/chef-workstation.rb

```diff
@@ -77,7 +87,7 @@
 override "zlib", version: "1.2.11"
 override "libzmq", version: "4.0.7"
 override "openssl", version: "1.0.2p"
-
+override "nodejs", version: "10.9.0"
```

Comment:
Need to remove this, holding off to avoid triggering a full rebuild until we're ready to land this.